frontend/src/pages/operation_list/new_operation_button/stylesheet.styl

```diff
@@ -22,3 +23,22 @@
   &:hover
     background: $lighter-background
     border-radius: 5px
+
+  // plus-inverse-size shrinks as the pixel size grows. max size is effective 9
+  $plus-inverse-size=13px
```

Comment:
this should be 14px to be centered. 13px results in it being 2px higher and to the right but otherwise looks great